src/renderers/common/Renderer.js

```diff
@@ -722,13 +722,7 @@ class Renderer {
 	 */
 	async init() {
 
-		if ( this._initialized ) {
-
-			throw new Error( 'Renderer: Backend has already been initialized.' );
-
-		}
-
-		if ( this._initPromise !== null ) {
+		if ( this._initialized === true || this._initPromise !== null ) {
```

Comment:
With this little trick, multiple components are allowed to call `init()` and work with the same init promise. See the changes in `Inspector` for further reference.